cumulus/parachains/integration-tests/emulated/common/src/macros.rs

```diff
@@ -127,8 +127,8 @@ macro_rules! include_penpal_create_foreign_asset_on_asset_hub {
 		$crate::impls::paste::paste! {
 			pub fn penpal_create_foreign_asset_on_asset_hub(
 				asset_id_on_penpal: u32,
-				foreign_asset_at_asset_hub: MultiLocation,
-				ah_as_seen_by_penpal: MultiLocation,
+				foreign_asset_at_asset_hub: v3::Location,
```

Comment:
Why is this specifically v3::Location?

Reply:
I'm leaving all storage keys in asset hub for foreign assets as `v3::Location`. I'll write a storage migration after this PR